source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,326 @@ Sorts Builders
+
+To combine sort criteria, use the ``Sorts.orderBy()`` static factory
+method. The ``orderBy()`` method builds sort criteria that apply passed 
+in sort criteria from left to right in the event of ties. 
+
+In the following code snippet, we use the ``orderBy()`` method to combine a
+descending sort on the ``letter`` field with an ascending sort on the
+``_id`` field.
+
+.. code-block:: java
+
+   import static com.mongodb.client.model.Sorts.orderBy;
+   import static com.mongodb.client.model.Sorts.ascending;
+   import static com.mongodb.client.model.Sorts.descending;
+
+   // <MongoCollection setup code here>
+
+   Bson orderBySort = orderBy(descending("letter"), ascending("_id"));
+   collection.find().sort(orderBySort);
+
+The code snippet above returns the documents in the
+:ref:`sort_example collection <sorts-builders-sort-example>`
+in the following order: 
+
+.. code-block:: json
+
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+
+Text Search
+-----------
+
+If you perform a text search on your collection with the
+:manual:`$text </reference/operator/query/text/>` query operator, you 
+can sort your results by how well they match your ``$search`` String using
+:manual:`text score </reference/operator/aggregation/meta/#exp._S_meta>`. 
+To sort your search results by the text scores associated with your text
+search, use the ``Sorts.metaTextScore()`` static factory method. 
+
+.. warning:: Make Sure to Create a Text Index
+
+   The ``$text`` query operator performs a text search on the content indexed 
+   by the :manual:`text index </core/index-text/>` of your collection.
+   Therefore, you must create a text index on your collection before you
+   can run a text search and use ``Sorts.metaTextScore()``.
+
+.. example::
+
+   In the following code example, we show how you can use the
+   ``Sorts.metaTextScore()`` method to sort the results of a text
+   search on the :ref:`sort_example collection <sorts-builders-sort-example>`.
+   The code example uses the :doc:`Filters </fundamentals/builders/filters>`,
+   :doc:`Indexes </fundamentals/builders/indexes>`, and
+   :doc:`Projections </fundamentals/builders/projections>` builders.
+   The code example performs the following actions:
+
+   #. Creates a text index for your
+      :ref:`sort_example collection <sorts-builders-sort-example>`
+      on the ``food`` field.
+   #. Runs your text search for the phrase "maple donut".
+   #. Projects ``Projections.metaTextScore()`` into your query results as the
+      ``score`` field. This projection is optional if your MongoDB instance is
+      running MongoDB 4.4 or later. 
+   #. Sorts using ``Sorts.metaTextScore()`` as your sort criteria.
```

Comment:
If possible, let's leave the code out of the description of the sequential actions. I think it's easier to describe without the full static method names, and the current descriptions probably need more detail to be precise.

Reply:
Thanks for the note. After implementing this change, I can see how it improves the quality of the list. This structure allows the prose to give a higher level description of code, rather than the prose restating what is in the code. 

Fix: https://github.com/mongodb/docs-java/pull/59/commits/ba9a57fe368df60e04e59051bd541f4af60523b0

Going forward: I will try to have prose provide alternative information to code rather than duplicating content of code.

Reply:
I would say you don't always have to leave the code out of it when you're pointing to a specific place where something happens, but it's preferable to avoid it.